data/presets/schema/field.json

```diff
@@ -59,6 +59,7 @@
                 "identifier",
                 "localized",
                 "manyCombo",
+                "maxheight",
```

Comment:
This PR will need to be reworked now that #8175 has split out the field schema into a separate [id-tagging-schema](https://github.com/openstreetmap/id-tagging-schema/) repository.

Reply:
Yep unfortunately that's the case. I can work on reconciling them.